email_alert_service/models/priority.rb

```diff
@@ -0,0 +1,33 @@
+class Priority
+  def initialize(document:)
```

Comment:
I don't think we add anything by having a keyword argument here since there's not likely to be ambiguity it's a document going in.